pkg/util/oom/oom.go

```diff
@@ -16,11 +16,16 @@ limitations under the License.
 
 package oom
 
+type Interface interface {
+	ApplyOOMScoreAdj(pid int, oomScoreAdj int) error
+	ApplyOOMScoreAdjContainer(cgroupName string, oomScoreAdj, maxTries int) error
+}
+
 // This is a struct instead of an interface to allow injection of process ID listers and
```

Comment:
Is this comment still accurate? Even if the answer is yes, could we slightly rephrase it, because to me, it feels like we are implying that we aren't going to create an interface.

Reply:
make sense, I'll fix it